Fix __title__ and doxygen \addtogroup blocks

luzpaz opened the pull request:
Still needs review

luzpaz commented on src/Mod/Fem/femsolver/elmer/equations/linear.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "_Linear"
+__title__ = "FreeCAD FEM _Linear"
```
should I remove the `_` from `_Linear` ?

berndhahnebach replied:
FreeCAD FEM solver elmer equation _Linear

luzpaz commented on src/Mod/Fem/femsolver/elmer/equations/nonlinear.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "_NonLinear"
+__title__ = "FreeCAD FEM _NonLinear"
```
Same question, remove `_` ?

berndhahnebach replied:
FreeCAD FEM solver elmer equation _NonLinear

luzpaz commented on src/Mod/Fem/femsolver/elmer/sifio.py:
```diff
@@ -20,6 +20,12 @@
 # *                                                                         *
 # ***************************************************************************
 
+__title__ = "FreeCAD FEM ???"
```
What is the correct `__title__`

berndhahnebach replied:
FreeCAD FEM solver elmer sifio

luzpaz commented on src/Mod/Fem/femsolver/elmer/writer.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "FemWriterElmer"
+__title__ = "FreeCAD FEM FemWriterElmer"
```
Please check the name

berndhahnebach replied:
FreeCAD FEM solver elmer writer

luzpaz commented on src/Mod/Fem/femsolver/equationbase.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "_Base"
+__title__ = "FreeCAD FEM _Base"
```
remove the `_` ?

berndhahnebach replied:
FreeCAD FEM solver elmer equation _Base

luzpaz commented on src/Mod/Fem/femsolver/reportdialog.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "reportdialog"
+__title__ = "FreeCAD FEM reportdialog"
```
`reportdialog` is ok? Or can it be `Report Dialog`?

berndhahnebach replied:
FreeCAD FEM solver report dialog

luzpaz commented on src/Mod/Fem/femsolver/run.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "run"
+__title__ = "FreeCAD FEM run"
```
`run` what ?

berndhahnebach replied:
FreeCAD FEM solver run

luzpaz commented on src/Mod/Fem/femsolver/settings.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "settings"
+__title__ = "FreeCAD FEM settings"
```
is `settings` correct? or can there be more elaboration ?

berndhahnebach replied:
FreeCAD FEM solver settings

luzpaz commented on src/Mod/Fem/femsolver/signal.py:
```diff
@@ -20,12 +20,15 @@
 
-
-__title__ = "signal"
+__title__ = "FreeCAD FEM signal"
```
is `signal` enough?

berndhahnebach replied:
FreeCAD FEM solver signal

luzpaz commented on src/Mod/Fem/femsolver/task.py:
```diff
@@ -20,11 +20,12 @@
 
-
-__title__ = "task"
+__title__ = "FreeCAD FEM task"
```
what `task` ?

berndhahnebach replied:
FreeCAD FEM solver task

luzpaz commented:
@berndhahnebach 
I resolved all the fixes you oriented me in -> 8d7dae0
I also found some other non-uniform things and fixed + added Fenics -> b487187

berndhahnebach commented:
I had a look at all modules in femsolver package. How about this?

See https://github.com/berndhahnebach/FreeCAD_bhb/commit/81be787523d20ed1828e3893ceb413597d0ab100

It is on top of your commits. If it fits for you too we can got for the other fem packages.

bernd

luzpaz committed:
FEM: solver, make file header title use the same schema in all solver modules 

Made uniform all __title__ doxy \addtogroup blocks

luzpaz commented:
FYI, since we're tackling the FEM header files, I figured we'd also make the copyright and developer info uniform as well. In eb28d5f you can see I just did it for `femsolver/` so far. 

I'm doing the same thing with [TechDraw WB](https://github.com/FreeCAD/FreeCAD/compare/master...WandererFan:TD-Header-Uniformity). I'm using the pattern that Werner uses when he creates a new page.
`Copyright (c) YEAR DevFirstName DevLastName <email@address.com>` 

Are you OK with that ?

luzpaz commented:
Sorry, @berndhahnebach but I didn't know how to give this latest commit a16ef0f as much love. Can you jump in. Im fact, I think you can actually manually make changes to the commit itself and push changes to it (since you're the admin of the repo)

berndhahnebach commented:
> I'm doing the same thing with [TechDraw WB](https://github.com/FreeCAD/FreeCAD/compare/master...WandererFan:TD-Header-Uniformity). I'm using the pattern that Werner uses when he creates a new page.
> `Copyright (c) YEAR DevFirstName DevLastName <email@address.com>`
> 
> Are you OK with that ?

of course!   It is what werner does and it is what is in the template of FreeCAD master: https://github.com/FreeCAD/FreeCAD/blob/master/src/Tools/_TEMPLATE_/App/PreCompiled.cpp#L2

cheers bernd

luzpaz committed:
FEM: solver, make uniform copyright and developer info 


.

berndhahnebach commented:
https://github.com/berndhahnebach/FreeCAD_bhb/commit/6633ee1419344ce54649d2e23c2246522987cccd

berndhahnebach commented:
either we fill ALL __init.py__ or we clear all of them ... I would clear them, how about you?

luzpaz commented:
> either we fill ALL **init.py** or we clear all of them ... I would clear them, how about you?

I'm cool with clearing them :+1:

berndhahnebach commented:
What's next?

luzpaz commented:
what branch do i rebase this now, `femtemp` or `femdev` ?

berndhahnebach commented:
best is to use femdev. femtmp is the branch I play with and cherry-pick all kind of code, and delete commtis and and and ...

berndhahnebach commented:
should I cheery-pick your commits and my into femdev for you ?

luzpaz commented:
no need. I will take care of it.

luzpaz committed:
FEM: make file headers title use the same schema in all fem(gui)objects/

Cherrypicked from femtemp f3212888c50d
+ 2 femobject uniformity fixes


fem(gui)objects leftover header copyright fix

berndhahnebach commented:
I added another two code formating commits ...

https://github.com/berndhahnebach/FreeCAD_bhb/commit/cb0a97ae4e2ece0e95e04e69ca84c691b5e36c2e
https://github.com/berndhahnebach/FreeCAD_bhb/commit/e9cfa953e4b8c65126c1f00528d47318e782adef

Would you like to continue or should I go for a PR?

luzpaz commented:
@berndhahnebach i think you should go to PR. The process is getting a little too murky. I think we made good progress and we can continue when needed. This will give me the opportunity to cleanup my git environment a bit (i have way too many stashed commits that I need to sort out). So green light from me.

berndhahnebach commented:
great work :-)

https://github.com/FreeCAD/FreeCAD/pull/1799

luzpaz commented:
after it's merged, can you please rebase femdev ? Thanks!

berndhahnebach commented:
FYI: Python does not like to ü in Jürgen ... https://github.com/FreeCAD/FreeCAD/commit/86455527ddb9eea15f5ae562c2f02b921d4c1a11

berndhahnebach commented:
https://github.com/berndhahnebach/FreeCAD_bhb/commits/femdev
